src/components/tools/WebSpeechAPIDemo/index.tsx

```
@@ -58,26 +58,20 @@ export const WebSpeechAPIDemo = () => {
-                'text-theme-subtitle',
+                'text-(--color-ink-3)',
                 'mb-16',
                 'text-center',
               )}
             >
               {getMessage()}
             </div>
 
-            <span dangerouslySetInnerHTML={{ __html: output }} />
-            <span className="opacity-50">{interimOutput}</span>
+            <span className="text-(--color-ink-2)" dangerouslySetInnerHTML={{ __html: output }} />
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)